packages/suite/src/views/settings/SettingsCoins/SettingsCoins.tsx

```diff
@@ -31,8 +31,9 @@ import { FirmwareTypeSuggestion } from './FirmwareTypeSuggestion';
 import { spacingsPx } from '@trezor/theme';
 import { selectSuiteFlags } from '../../../reducers/suite/suiteReducer';
 
-const StyledButton = styled(Button)`
```

Comment:
this was eslint warning for overloading, so I remade it as wrapper;
I needed a wrapper anyway for the `width: fit-content` (so that the tooltip is positioned correctly)

Reply:
Nice! Thank you for refactor overriding 🙏